aclogview/CM_Movement.cs

```diff
@@ -664,6 +669,44 @@ public class Jump_NonAutonomous : Message {
         }
     }
 
+    public class LifestoneMaterialize : Message
+    {
+        public int ObjectId;
+        public Position Position;
+        public uint unknown2;
+        public uint unknown3;
+        public uint unknown4;
+        public uint unknown5;
+
+        public static LifestoneMaterialize read(BinaryReader binaryReader)
+        {
+            LifestoneMaterialize newObj = new LifestoneMaterialize();
+            newObj.ObjectId = binaryReader.ReadInt32();
+            newObj.Position = Position.read(binaryReader);
```

Comment:
Looks like there are 4 unknown bytes between the object id and the position. It also looks like there are at least 24 bytes following the position. Suggest adding an unknown int between id and position as well as adding two more unknown ints to the end to fully read the packet.